rero_ils/modules/collections/jsonschemas/collections/collection-v0.0.1.json

```diff
@@ -0,0 +1,250 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "Collection",
+  "description": "JSON schema for collection.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "collection_id",
+    "title",
+    "collection_type",
+    "start_date",
+    "end_date",
+    "organisation",
+    "published"
+  ],
+  "propertiesOrder": [
+    "collection_type",
+    "start_date",
+    "end_date",
+    "title",
+    "collection_id",
+    "teachers",
+    "description",
+    "subjects",
+    "teachers",
+    "libraries",
+    "items",
+    "published"
+  ],
+  "properties": {
+    "$schema": {
+      "title": "Schema",
+      "description": "Schema to validate collection records against.",
+      "type": "string",
+      "minLength": 9,
+      "default": "https://ils.rero.ch/schema/collections/collection-v0.0.1.json"
+    },
+    "pid": {
+      "title": "Internal collection PID",
+      "type": "string",
+      "minLength": 1
+    },
+    "collection_id": {
+      "title": "Local identifier of the collection",
+      "type": "string",
+      "minLength": 1
+    },
+    "title": {
+      "title": "Title of the Collection",
+      "type": "string",
+      "minLength": 1
+    },
+    "collection_type": {
+      "title": "Type",
+      "type": "string",
+      "enum": [
+        "course",
+        "exhibition"
+      ],
+      "form": {
+        "type": "selectWithSort",
+        "wrappers": [
+          "form-field"
+        ],
+        "options": [
+          {
+            "label": "course",
+            "value": "course"
+          },
+          {
+            "label": "exhibition",
+            "value": "exhibition"
+          }
+        ],
+        "templateOptions": {
+          "selectWithSortOptions": {
+            "order": "label"
+          }
+        }
+      }
+    },
+    "teachers": {
```

Comment:
I understand this will be a $ref for a patron record in the future.

Reply:
Yes, @benerken asked an array of strings for the moment.